cdm/src/main/java/ucar/nc2/ft2/simpgeometry/CFLine.java

```diff
@@ -277,6 +305,9 @@ public Line setupLine(NetcdfDataset dataset, Variable var, int index)
 		} catch (InvalidRangeException e) {
 			e.printStackTrace();
 			return null;
+		} catch (InvalidDataseriesException e) {

```

Comment:
Oh yes of course, will use multicatching!

Reply:
And with regards to SLF4J, that seems to be another ticket, so perhaps it would be good to change it stackTrace to SLF4J in another ticket?

Reply:
It would definitely be out of this task's scope to replace all calls to Exception.printStackTrace/System.out.println/System.err.println with SLF4J calls in all of the classes we created this summer.  To switch to SLF4J in just the spot I commented I think all you would need are the exact two lines I included in the comment above, and an IDE-autogenerated import statement or two, so it should be trivially cheap to implement. If it is not that easy, no need to adopt now.